docs/docs/01-index.md

```diff
@@ -18,10 +19,9 @@ For more granular packages we recommend reading the documentation.
 # Features
 
 - **General**
-    - [Securely Receive CloudEvents](features/cloudevent/receive-cloudevents-job)
+    - [Securely Receive CloudEvents](features/general/receive-cloudevents-job)
 - **Security**
     - [Automatically invalidate cached secrets from Azure Key Vault](features/security/auto-invalidate-secrets)
-    - [Automatically restart Azure Service Bus message pump on rotated credentials](features/security/auto-restart-servicebus-messagepump-on-rotated-credentials)
```

Comment:
Is this on purpose or did you forget to merge master because this is something that is yet-to-be-released?

Reply:
Yes, I think the content of the `./docs/preview` should become `./docs` in this setup?

Reply:
Correct, `./docs` folder is meant to be used as the `next` release. When this is ready to go live, simply run the versioning script or if you don't want a new version. Copy the files to the desired existing version directory.